src/frontend/services/cookie-service.js

```diff
@@ -3,38 +3,58 @@ const PASSWORD_COOKIE_NAME = "password";
 const HEADER_COOKIE_NAME = "header";
 const TOKEN_COOKIE_NAME = "token";
 
-function getCookiePropertyByName(name) {
-  const result = document.cookie
-    .split("; ")
-    .find((row) => row.startsWith(`${name}=`))
-    ?.split("=")[1];
-
-  return result;
-}
-
 export class CookieService {
-  static getCookie() {
-    let isValid = true;
-    const user = getCookiePropertyByName(USERNAME_COOKIE_NAME);
-    if (!user) isValid = false;
-
-    const pass = getCookiePropertyByName(PASSWORD_COOKIE_NAME);
-    if (!pass) isValid = false;
-
-    const header = getCookiePropertyByName(HEADER_COOKIE_NAME);
-    if (!header) isValid = false;
-
-    const token = getCookiePropertyByName(TOKEN_COOKIE_NAME);
-    if (!token) isValid = false;
-
-    let cookie = {
-      username: user,
-      password: pass,
-      token: token,
-      header: header,
-      isValid: isValid,
-    };
-
-    return cookie;
-  }
+	static COOKIE_NAMES = {
+		USERNAME: "username",
+		PASSWORD: "password",
+		HEADER: "header",
+		TOKEN: "token",
+		LAST_CHAT: "last-chat",
+	};
+
+	static getCookieByKey(name) {
+		const result = document.cookie
+			.split("; ")
+			.find((row) => row.startsWith(`${name}=`))
+			?.split("=")[1];
+
+		return result;
+	}
+	static getLoginCookies() {
```

Comment:
il mome non mi convince:
- il cookie è uno solo 
- che sia il coockie di login è indicato solo in questo nome di funzione